web/src/global/analytics.js

```diff
@@ -0,0 +1,18 @@
+import Analytics from "analytics"
+import googleAnalytics from "@analytics/google-analytics"
+import mixpanelPlugin from "@analytics/mixpanel"
+import publicConfig from "src/global/publicConfig"
+
+const analytics = Analytics({
+  app: "kitty-items",
+  plugins: [
+    googleAnalytics({
+      trackingId: publicConfig.gaTrackingId,
+    }),
+    mixpanelPlugin({
+      token: publicConfig.mixpanelToken,
+    }),
+  ],
+})
```

Comment:
Does this fail if `gaTrackingId` or `mixpanelToken` or not set? As mentioned above, would be nice to make these optional for users.